draft-yasskin-http-origin-signed-responses.md

```diff
@@ -959,6 +959,16 @@ extension. This OID might or might not be used as the final OID for the
 extension, so certificates including it might need to be reissued once the final
 RFC is published.
 
+### Extensions to the CAA Record: cansignhttpexchanges Parameter {#caa-cansignhttpexchanges}
+
+A CAA parameter "cansignhttpexchanges" is defined for the "issue" and
+"issuewild" properties defined by {{!RFC6844}}.  The value of this parameter, if
+specified, SHOULD be "yes".
+
+If the "cansignhttpexchanges" parameter is not present and equal to "yes", the
+CA indicated by the "issue" or "issuewild" property SHOULD NOT issue certificates
```

Comment:
Am I correct that this only applies to CA who issue certificates via ACME protocol?

Reply:
No. All CAs are required to check for and respect CAA (if present) by the Baseline Requirements.

This change would require that site operators wishing to obtain a CanSignHttpExchanges certificate take affirmative steps to modify their DNS to indicate consent. Combined with the above requirement, this spec change has the effect of declaring how all CAs must obtain consent to issue these certificates, and declaring the means by which site operators can deny consent. It favors 'defenders' (site operators at risk from a CA using a less-than-ideal validation method or being tricked temporarily) by requiring consent by opt-in, rather than opt-out.